### ISSUE TITLE
<amp-date-picker>: locale support converts numbers to Devanagari language

### ISSUE THREAD

**Issue by akanshgulati** [ampproject/amphtml#28045]
## Currently locale support in amp-date-picker converts the date from English letters to Devanagari language which is yet not understandable by large users. 

It should have a modular level locale system or more customisation to only convert "months" and "week days" as particular locale i.e. "hi" and do not change the numbers in that format. 

## Additional context

Screenshot of Devanagari numbers coming up

<img width="314" alt="Screenshot 2020-04-27 at 14 33 08" src="https://user-images.githubusercontent.com/5287100/80354468-69d00800-8894-11ea-8b97-e802ed9faf0b.png">

**Comment by caroqliu** [comment]
Thanks for filing the issue! I'm not sure I understand -- what are you setting as the `locale` attribute on the component? It appears as though the expected result is English but the result is the screenshot.

**Comment by akanshgulati** [comment]
@caroqliu We are setting **"hi"** as locale attribute value. So my concern it that the letters coming with the "hi" locale is in Devanagari language, i.e. `१` coming for digit `1` as in English. I want to request that in `amp-date-display` digits should come as English digit instead because currently Devanagari, १ letters are not understandable by much of the crowd even after using `hi` locale value. 

Apart from digits, all things should be same as coming in above attached screenshot. 

Sharing the digits comparison here: 
<img width="368" alt="Screenshot 2020-04-29 at 00 56 06" src="https://user-images.githubusercontent.com/5287100/80529120-9b3ff500-89b4-11ea-82bf-bc5d47c51ef4.png">

**Comment by caroqliu** [comment]
Thanks for describing the issue further. Unfortunately the `amp-date-picker` relies on [moment.js](https://momentjs.com/) for locale support and we cannot special case this value at the component layer. It looks the [moment.js implementation](https://github.com/moment/moment/issues/383#issuecomment-7764248) describes these numerals to be working as intended in Devanagari.

Closing the issue now, though I'd recommend if you intend to pursue this further to file an issue with @moment/moment
